src/session/content/message_row/bubble.rs

```diff
@@ -40,6 +40,10 @@ mod imp {
                     visible: false;
                 }
 
+                .MessageReply message_reply {
```

Comment:
Having an entire `MessageReply` widget for each bubble even when they aren't replying to a message is not that great. Can you change this to an `AdwBin` and then add the `MessageReply` as its child only when you need it? Basically just like the `prefix_bin` below.